pkg/integrations/v4/supervisor_fb_conf_linux.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//go:build linux
+
+package v4
+
+import "path/filepath"
+
+const (
+	// defaults for td-agent-bit (<=1.9).
+	defaultLoggingBinDir1 = "/opt/td-agent-bit/bin"
+	defaultFluentBitExe1  = "td-agent-bit"
```

Comment:
This name is a bit confusing with `.exe` for windows, maybe defaultFluentBitExec1 or defaultFluentBitExecutable1

Reply:
changed in c732cd1e33490d7a1bbd08640394eb9533913393